### PR TITLE
Implement PDF Uplaod

### PR DESCRIPTION
**Pull request by retato-snu**
1. PDF to text
2. remove line break
3. translate text

Can upload multiple files & can download only after preprocessing(remove line break)

### CODE REVIEW

File: easyword_translator/pdf.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import fitz  # PyMuPDF
```
Comment (Zeta611) [inline easyword_translator/pdf.py]: pyproject.toml 파일에  dependency로 추가 후,
```sh
poetry export -f requirements.txt --output requirements.txt
```
로 requirements.txt로 업데이트해야 합니다!

File: easyword_translator/run.py [hunk 2]
```diff
@@ -150,6 +234,24 @@ def translate(sentence: str) -> str:
             title=TITLE,
             description=DESCRIPTION,
         )
+    with gr.Tab("PDF"):
+        pdf_file = PDFFile()
+        upload_button = gr.UploadButton(
+            label="Upload PDF",
+            file_types=[".pdf"],
+        )
+        uplaod_file_list_box = gr.File(label="Uploaded Files")
```
Comment (Zeta611) [inline easyword_translator/run.py]: 오타 있어요 `uplaod_...` -> `upload_...`

**Review by Zeta611 — CHANGES_REQUESTED** [review]

File: easyword_translator/pdf.py (hunk 1 above)
Comment (retato-snu) [reply easyword_translator/pdf.py]: PyMuPDF가 requirements에 있어서 상관없을것같긴한데...

Comment (retato-snu) [reply easyword_translator/pdf.py]: pymupdf로 바꿨어요

**Comment by retato-snu** [comment]
> 1. PDF to text
> 2. remove line break
> 3. translate text
> 
> Can upload multiple files & can download only after preprocessing(remove line break)

**Review by Zeta611 — APPROVED** [review]

**Comment by Zeta611** [comment]
LGTM!

**Pull request merged**